static/css/components/edit-toolbar.less

```diff
@@ -10,19 +10,29 @@ div#editHistory {
   width: auto;
 }
 
+div#editButton {
+  float: left;
+}
+
 div#editInfo {
+  float: right;
+  overflow: hidden;
   white-space: nowrap;
   text-align: right;
   > div {
     font-size: 11px;
     font-weight: normal;
     color: @brown!important;
     font-family: @lucida_sans_serif-1;
+    + div {
```

Comment:
this is exceeding one of our css linting rules (depth >2)

Reply:
Hm. I may have to modify the markup to avoid this, then. I don't really like adding the padding on the opposite side and targeting the `:last-of-type` instead, especially with a generic element like a div.

Reply:
Updated the selector. There's still a failure in the tests from when I first branched off of master.

Reply:
You can also consider adding new styles using BEM notation. I've been slowly doing that in other places..

Reply:
I'm personally not a fan of BEM because it tends to move the style language to the markup. I noticed there was already a class on the element I wanted to style, so I just attached to that.

Reply:
I think having a space for discussing these types of tradeoffs would be useful BEM v. not seems like a useful community conversation for a call. I generally defer to @jdlrobson's judgement as our deputy of front-end, though I take this as a suggestion rather than a mandate. It would be nice to make progress on css patterns and reach some consensus.

I don't think this issue should be blocked / held hostage by it, especially given a component existed.

Reply:
Definitely not a blocker. Only blocker is the padding suggestion and necessary rebase. @shaneriley any thoughts on that? Also happy to submit this myself in a follow-up patch if you don't have time :-)

Reply:
Should be all set now. I've removed the top padding and rebased master onto the branch.